packages/core/addon/components/navi-visualizations/bar-chart.js

```diff
@@ -27,6 +27,6 @@ export default LineChart.extend({
    * @property {String} c3ChartType - c3 chart type
    */
   c3ChartType: computed('chartType', function() {
-    return this.chartType;
+    return get(this, 'chartType');
```

Comment:
I think it only affects cp's so `this.options` or `this.chartType` would still work